src/components/language-select/language-toggle/template.njk

```diff
@@ -0,0 +1,15 @@
+<nav class="language-select {{params.class}}" aria-label="{{params.ariaLabel}}">
```

Comment:
The aria-label hear isn't being read out by the screen reader, when the language selection is focused it reads out the scope of the link and as there's no specific label associated with that element the output is `link, {{language}}, list 2 items`. We can add the aria-label to the anchor tag to provide a more descriptive output, or hide a label using a class to move it from the page

Reply:
Here, for the nav tag, Aria Label is a brief description of the purpose of the navigation, omitting the term "navigation", as the screen reader will read both the role and the contents of the label.
(More info: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/navigation_role)

I believe it's not mandatory/useful to add the aria-label property at the anchor level because the text in the link ("English" f.e) is clear enough. But tell me if I'm wrong and I will add it.